frontend/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "fam-frontend",
   "version": "0.0.0",
   "scripts": {
+    "install-frontend": "cd ../client-code-gen/gen && npm install && cd ../../frontend && npm ci",
```

Comment:
Why don't you use 'npm ci' in client-code-gen directory as well?

Reply:
Use ci now, had that thought too but forgot to change.